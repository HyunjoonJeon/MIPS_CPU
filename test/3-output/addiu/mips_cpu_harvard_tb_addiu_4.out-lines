register_v0:8b08a1fa
