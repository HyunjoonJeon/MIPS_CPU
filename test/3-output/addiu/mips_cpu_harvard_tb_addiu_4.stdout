loading instruction mem with test/1-binary/addiu/addiu_4.hex.txt
WARNING: rtl/mem_harvard.v:51: $readmemh: Standard inconsistency, following 1364-2005.
WARNING: rtl/mem_harvard.v:51: $readmemh(test/1-binary/addiu/addiu_4.hex.txt): Not enough words in the file for the requested range [0:2047].

TB : finished; active=0
register_v0:8b08a1fa
